tests/kafkatest/tests/core/consume_bench_test.py

```diff
@@ -91,11 +99,17 @@ def test_consume_bench(self, topics, metadata_quorum=quorum.zk):
         self.logger.info("TASKS: %s\n" % json.dumps(tasks, sort_keys=True, indent=2))
 
     @cluster(num_nodes=10)
-    @matrix(metadata_quorum=quorum.all_non_upgrade)
-    def test_single_partition(self, metadata_quorum=quorum.zk):
+    @matrix(
+        metadata_quorum=quorum.all_non_upgrade,
+        use_new_coordinator=[True, False]
+    )
```

Comment:
same applies here:
```suggestion
    @matrix(
        metadata_quorum=[quorum.isolated_kraft],
        use_new_coordinator=[True, False]
    )
    @matrix(
        metadata_quorum=[quorum.zk],
        use_new_coordinator=[False]
    )
```

There are some alternitives, one could be your `skip_if_new_coordinator_and_zk` wrapper could go into the method metadata, and remove the testcases with use_new_coordinator=True and metadata_quorum=zk, though id advise against it as its not that simple.